cfgov/unprocessed/css/pages/event.less

```diff
@@ -128,8 +128,15 @@
 .event-meta_address {
     text-transform: capitalize;
     display: inline-block;
+
+    .event-meta_street:not(:empty):after {
+        content: '\A';
```

Comment:
It just add a line break. Fixed in https://github.com/cfpb/cfgov-refresh/pull/1841/commits/067b69a782784f78a8f11dfc0a27f771bd39d2af